styles/styles.css

```diff
@@ -0,0 +1,89 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  width: 100%;
+}
+
+.container {
+  padding: 120px 104px;
+  margin: 0 auto;
+}
+
+.advantages {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  gap: 50px;
+  text-align: center;
+  align-items: flex-end;
+  margin: 0 auto;
+}
+
+.col {
+  width: 33%;
+  height: 178px;
+}
+
+.advantages--row {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.advantages--row__headphones {
+  margin-bottom: 52px;
```

Comment:
These margins look weird. Better to add wrappers for your images to have opportunity to align them as you need. Why did you miss it from the design?